apps/rush-lib/src/utilities/VersionControl.ts

```diff
@@ -48,49 +66,50 @@ export class VersionControl {
    * @param repositoryUrl - repository url
    */
   public static getRemoteMasterBranch(repositoryUrl?: string): string {
-    const defaultRemote: string = 'origin';
-    const defaultMaster: string = 'origin/master';
-    let useDefault: boolean = false;
+    const defaultMaster: string = `${DEFAULT_REMOTE}/${DEFAULT_BRANCH}`;
     let matchingRemotes: string[] = [];
 
-    if (!repositoryUrl) {
-      useDefault = true;
-    } else {
+    if (repositoryUrl) {
       const output: string = child_process
-      .execSync(`git remote`)
-      .toString();
+        .execSync(`git remote`)
+        .toString();
       matchingRemotes = output.split('\n').filter(remoteName => {
         if (remoteName) {
           const remoteUrl: string = child_process.execSync(`git remote get-url ${remoteName}`)
             .toString()
             .trim();
-          if (remoteName === defaultRemote && remoteUrl === repositoryUrl) {
-            useDefault = true;
-          }
           return remoteUrl === repositoryUrl;
         }
         return false;
       });
+    } else {
+      console.log(colors.yellow(
+        'A remote URL has not been specified in rush.json. Setting the baseline remote URL is recommended.'
```

Comment:
Would we want to disable this warning in unforkable repos like **sp-client**?

Or should we insist that `repositoryUrl` be defined for any repo that's using `rush change`, regardless of whether people fork?

Reply:
I think we should just require `repositoryUrl`.